### PR TITLE
Object types categories

### PR DESCRIPTION
**Pull request by OKT90**

### CODE REVIEW

File: Idoit.API.Client/CMDB/ObjectTypeCategories/ObjectTypeCategories.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+namespace Idoit.API.Client.CMDB.ObjectTypeCategories
+{
+    public class ObjectTypeCategories
+    {
+        public ObjectTypeCategories(Client myClient)
+        {
+            client = myClient;
+        }
+        Dictionary<string, object> parameter;
```
Comment (yhildebrandt) [inline Idoit.API.Client/CMDB/ObjectTypeCategories/ObjectTypeCategories.cs]: Again: Class properties above constructor?

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        string URL;
+        string APIKEY;
+        string LANGUAGE;
+        public ObjectTypeCategoriesTest()
+        {
+            string path = Path.Combine(Environment.CurrentDirectory, @"Data\", "Api.env");
+            DotNetEnv.Env.Load(path); URL = DotNetEnv.Env.GetString("URL");
+            APIKEY = DotNetEnv.Env.GetString("APIKEY");
+            LANGUAGE = DotNetEnv.Env.GetString("LANGUAGE");
+        }
```
Comment (mntksp) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: Again, what does this test? There is no assertion here.

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        //Read
+        [TestMethod]
+        public void ReadTest()
+        {
+            string costumId = "";
```
Comment (mntksp) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: custom

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+            string objectTypeId = "";
+            List<Result[]> lists = new List<Result[]>();
```
Comment (mntksp) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: "lists" is a very general name. Is it possible to give this a more speaking name?

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs (hunk 2 above)
Comment (mntksp) [reply UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: Still the same, this should be a setup class, if I'm not mistaken.

**Review by yhildebrandt — COMMENTED** [review]
@OKT90  Take a look at the comments

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 5]
```diff
@@ -29,9 +29,9 @@ public ObjectTypeCategoriesTest()
         [TestMethod]
         public void ReadTest()
         {
-            string costumId = "";
+            string customId = "";
```
Comment (yhildebrandt) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: Renamed customId to customId? 🤔

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 6]
```diff
@@ -59,10 +59,10 @@ public void ReadTest()
             {
                 if (element.title == "Diary")
                 {
-                    costumId = element.id;
+                    customId = element.id;
```
Comment (yhildebrandt) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: Same in this line, I think you wanted to rename it to "custom"

File: UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs [hunk 7]
```diff
@@ -59,10 +59,10 @@ public void ReadTest()
...
                 }
             }
-            Assert.IsNotNull(costumId);
+            Assert.IsNotNull(customId);
```
Comment (yhildebrandt) [inline UnitTestApi/CMDB/ObjectTypeCategories/ObjectTypeCategoriesTest.cs]: Same here

**Review by helbow — APPROVED** [review]

**Review by yhildebrandt — APPROVED** [review]

**Pull request merged**